exercises/C-variables/exercise.js

```diff
@@ -1,3 +1,3 @@
-// Start by creating a variable `greeting`
-
-console.log(greeting);
+//* Print your `greeting` to the console 3 times
+let greeting = "Welcome to coding world";
+for (let i = 0; i < 3; i++) console.log(greeting);
```

Comment:
At this stage there is no need to have a 'for' loop, it also won't answer the question as the code is not adding anything. To have three 'console.log()' , you just need write them like this:
console.log(greeting);
console.log(greeting);
console.log(greeting);